### PR TITLE
create byline brand component

### PR DESCRIPTION
**Pull request by rogervera**
## Create a simple Byline component for the Brand Package

Will be used in creating the Learner Testimonials component, as well as in the Learner Story pages.

Screen Shot 2020-02-20 at 11 39 15 AM
Screen Shot 2020-02-20 at 11 39 21 AM
Screen Shot 2020-02-20 at 11 47 21 AM
Screen Shot 2020-02-20 at 11 47 07 AM
---
#### The location icon pin differs from design, and design suggests to use  https://drive.google.com/file/d/11Euwakgd8onnqY7BBgBmPqp8GCN8X0E0/view since access to the design icon pin is pending. https://codecademy.slack.com/archives/C6E64E3QS/p1582140538000600?thread_ts=1578424325.000200&cid=C6E64E3QS
---

## Merging your changes

When you are ready to merge to master, use the "squash and merge" button in GitHub, and follow the [commit message guide](/README.md#commit-message-guide) to write your commit message.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.1.1-alpha.4e06d3.4+4e06d3db<br/></pre>

File: packages/brand-components/src/Byline/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <span aria-label="Name" className={cx(s.name, classNames.name)}>
+      {name}
+    </span>
+    <span aria-label="Occupation" className={classNames.occupation}>
+      {occupation}
+    </span>
+    {location && (
+      <div className={s.locationContainer}>
+        <img alt="" className={s.networkPin} src={networkPin} />
+        <span aria-label="Location" className={classNames.location}>
```
Comment (JoshuaKGoldberg) [inline packages/brand-components/src/Byline/index.tsx]: Setting the `aria-label` here manually means the location will be announced as just Location, I think. It'd be better to give an informative `alt` to the `<img`, like `Location pin icon`.

File: packages/styleguide/stories/Byline.stories.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+export default {
+  component: Byline,
+  title: 'Brand/Byline',
+  decorators: [withKnobs],
+};
+
+export const byline = () => (
+  <Byline
+    name={text('Name', 'Bill Murray')}
+    occupation={text('Occupation', 'Ghostbuster @ Ghostcademy')}
```
Comment (JoshuaKGoldberg) [inline packages/styleguide/stories/Byline.stories.js]: _I love it 👻_

File: packages/brand-components/src/Byline/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
+@import "~@codecademy/gamut-styles/utils";
+
+.bylineContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.name {
+  font-size: 18px;
```
Comment (jakemhiller) [inline packages/brand-components/src/Byline/styles.module.scss]: we should still stick to `rems` in these components if possible

Comment (rogervera) [reply packages/brand-components/src/Byline/styles.module.scss]: Do you mean on other properties like `width`, `height`, `margin*`?
I thought `font-size` scales regardless. 

Or is this only applicable in our monolith?
https://codecademy.slack.com/archives/C047ESP1T/p1548881088106500

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.1.1-alpha.4aca49.7+4aca4944<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.1.2-alpha.e60fd6.8+e60fd689<br/>@codecademy/gamut-templates@2.2.33-alpha.e60fd6.8+e60fd689<br/>@codecademy/gamut@2.4.9-alpha.e60fd6.8+e60fd689<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.1.2-alpha.429dca.10+429dca28<br/></pre>

**Pull request merged**